src/views/Home.jsx

```diff
@@ -1,11 +1,15 @@
 import './Home.css';
+import { Link } from 'react-router-dom';
 
-export function Home() {
+export function Home({ handleNewToken }) {
 	return (
 		<div className="Home">
 			<p>
 				Hello from the home (<code>/</code>) page!
 			</p>
+			<Link to="/list">
+				<button onClick={handleNewToken}>Create a New List</button>
+			</Link>
```

Comment:
I'm wondering if instead of wrapping this button with <Link/> we could incorporate it within the handleNewToken function? Perhaps after a listToken is saved you can use Navigate to navigate to the /list route?

Reply:
We are open to learning other methods. no changes made for now

Reply:
You don't need a button at all...  you could just have the text link... which is a stylistic UX thing and has accessibility discussions either way...